crates/matrix-sdk-ui/src/timeline/event_item/mod.rs

```diff
@@ -335,6 +338,18 @@ impl EventTimelineItem {
         }
     }
 
+    /// Gets the `ShieldState` which can be used to decorate messages in the
```

Comment:
Auto-links are nice and cheap.

```suggestion
    /// Gets the [`ShieldState`] which can be used to decorate messages in the
```